Makefile

```diff
@@ -70,12 +112,14 @@ check:
 	  --format $(OPA_FORMAT) \
 	  $(OPA_QUERY)
 
+#--------------------------------------------------------------------
+
 OPA_VER=v0.39.0
 OPA_FILE=opa_linux_amd64_static
 OPA_URL=https://openpolicyagent.org/downloads/$(OPA_VER)/$(OPA_FILE)
 OPA_SHA=19a24f51d954190c02aafeac5867c9add286c6ab12ea85b3d8d348c98d633319
 ifndef OPA_BIN
-	OPA_BIN=$(HOME)/bin
+  OPA_BIN=$(HOME)/bin
```

Comment:
Intentional. Line 72 didn't work with a tab. Tabs are weird in make I guess? So I switched to spaces here to be consistent with that.